core/domain/classifier_services.py

```diff
@@ -618,24 +686,38 @@ def get_classifier_training_job_maps(exp_id, exp_version, state_names):
             mappings.
 
     Returns:
-        list(dict(str: str)). A list of dicts, each mapping
+        list(dict(str: str)|None). A list of dicts, each mapping
         algorithm IDs to the corresponding job IDs. Each element
         in the list corresponds to the corresponding state name in the
-        state_names input argument.
+        state_names input argument. None, if no StateMappingModel exists
+        for corresponding exploration id.
     """
     state_training_jobs_mapping_models = (
         classifier_models.StateTrainingJobsMappingModel.get_models(
             exp_id, exp_version, state_names))
-    state_to_algorithm_id_job_id_maps = []
+    state_to_algorithm_id_job_id_maps: List[Optional[Dict[str, str]]] = []
     for state_mapping_model in state_training_jobs_mapping_models:
-        state_to_algorithm_id_job_id_maps.append(
-            None if state_mapping_model is None
-            else state_mapping_model.algorithm_ids_to_job_ids)
+        if state_mapping_model:
+            # TODO(#15621): The explicit declaration of type for ndb properties
```

Comment:
I don't really understand how this comment relates to the code that follows it. Could you explain a bit more?

Reply:
Here, `state_mapping_model.algorithm_ids_to_job_ids` is an instance of ndb's JsonProperty. And currently in our stubs, all the ndb properties are annotated as `Any` type.  So that's why I explicitly added type annotation and TODO here